locales/en-US/app.ftl

```diff
@@ -776,42 +776,48 @@ TrackPower--tooltip-power-milliwatt = { $value } mW
 # watt-hour unit.
 # Variables:
 #   $value (String) - the energy value for this range
-TrackPower--tooltip-energy-used-in-range-watthour = { $value } Wh
+#   $carbonValue (string) - the equivalent carbon value (grams)
```

Comment:
> @flodolo do you mean explain it in the visible tooltip, or just as a comment with the variable?

Comment to the string for the variable, because there was a risk people wouldn't understand `co2e`. I think it's less likely now, but still won't hurt adding it.

Reply:
I'm adding something to the main comment for the TrackPower, I'll let you all have a look at it before merging.

Reply:
what do you think? https://github.com/firefox-devtools/profiler/pull/4372/commits/7062d6315b08ab22bdb8ce2318d6aede38445892

Reply:
Looks good to me.